dashboard-for-web/articles/web-dashboard-designer-mode/designing-dashboard-items/scatter-chart.md

```diff
@@ -26,4 +26,7 @@ This section is divided into the following subsections.
 	Describes how to toggle the chart's orientation.
 * [Labels](scatter-chart/labels.md)
 	
```

Comment:
Please change the following lines:

Line 7:
`The topics in this section describe the features available in the **Scatter Chart** dashboard item, and explain how to create and customize scatter charts in the Web Dashboard.`

Line 14:
`Explains how to supply the Scatter Chart dashboard item with data.`

Line 20:
`Describes the chart legend and its options.`